games/devtest/mods/testnodes/properties.lua

```diff
@@ -588,3 +588,36 @@ minetest.register_node("testnodes:drowning_1", {
 	groups = {dig_immediate=3},
 })
 
+-- post_effect_color_shaded
+
+minetest.register_node("testnodes:post_effect_color_shaded_false", {
+	description = S("\"post_effect_color_shaded = false\" Node"),
+
+	drawtype = "allfaces",
+	tiles = {"testnodes_post_effect_color_shaded_false.png"},
```

Comment:
Missing textures.

Reply:
Oops, forgot to `git add` them... Fixed.